rllib/algorithms/appo/appo.py

```diff
@@ -234,14 +234,13 @@ def get_default_rl_module_spec(self) -> SingleAgentRLModuleSpec:
 
     @override(ImpalaConfig)
     def get_learner_hyperparameters(self) -> AppoHyperparameters:
-        base_hps = super().get_learner_hyperparameters()
         return AppoHyperparameters(
             use_kl_loss=self.use_kl_loss,
             kl_target=self.kl_target,
             kl_coeff=self.kl_coeff,
             clip_param=self.clip_param,
             tau=self.tau,
-            **dataclasses.asdict(base_hps),
+            **dataclasses.asdict(super().get_learner_hyperparameters()),
```

Comment:
super nit: I think the previous code was more readable.

Reply:
fixed.